sihl_contrib/session/lib/model.ml

```diff
@@ -1,3 +1,8 @@
 module Session = struct
-  type t = { id : string; data : string; expire_date : string }
+  type t = { key : string; data : string; expire_date : Ptime.t }
+
+  let create () =
+    { key = "TODO"; data = "{}"; expire_date = Ptime_clock.now () }
```

Comment:
Clock should be provided as a part of core services so that it can be mocked for testing.

Reply:
I inject Ptime_clock now as dependency through function arguments for now to make the functions pure.